src/components/Heading/Heading.stories.tsx

```diff
@@ -58,7 +58,8 @@ export const Heading6: StoryObj<Args> = {
 export const Variants: StoryObj<Args> = {
   ...Heading1,
   render: (args) => {
-    const headings = [];
+    // `typeof Heading` doesn't work, but `any` typing is fine here
+    const headings: Array<any> = [];
```

Comment:
I'm guessing `typeof Heading` gets us a Component type, not a rendered element/JSX type.

Something like `const headings: Array<JSX.Element> = []` should work.

Can also see that by hovering over `headings.push` and seeing what type TS thinks is getting pushed onto the array.

Reply:
sweet, thanks!